src/app/edit-resource/edit-resource.component.html

```diff
@@ -0,0 +1,31 @@
+  <div class="modal-dialog" role="document">
```

Comment:
What is the difference between edit and create resource components? Can they be merged or generalized?